src/nixpacks/app.rs

```diff
@@ -26,12 +30,48 @@ impl App {
         fs::canonicalize(self.source.join(name)).is_ok()
     }
 
+    pub fn find_files(&self, pattern: &str) -> Result<Vec<String>> {
+        let full_pattern = self.source.join(pattern);
+
+        let pattern_str = match full_pattern.to_str() {
+            Some(s) => s,
+            None => return Ok(Vec::new()),
+        };
+
+        let relative_paths = glob(pattern_str)?
+            .filter_map(|p| p.ok()) // Remove bad ones
```

Comment:
Should this be a filter map or just a filter?

Reply:
Looks to be a filter. The function is being used wrong here so it makes sense.

Reply:
`filter_map` is correct. It needs to map in order to convert the `Option` from `p.ok()` into a `PathBuf`

Reply:
Ah, thought it was `is_ok`. My bad! 😅